### PR TITLE
Get the list of transactions for a given block

### PR DESCRIPTION
**Pull request by NemTechCN**
Get the list of transactions for a given block and fix bug transaction.message.payload.length undefined

### CODE REVIEW

File: src/service/transaction.service.ts [hunk 1]
```diff
@@ -51,7 +51,11 @@ export class TransactionService {
             } else {
                 transactionFormatted += transaction.recipient.toHex();
             }
-            transactionFormatted += transaction.message.payload.length > 0 ? ' Message:\"' + transaction.message.payload + '\"' : '';
+            try {
```
Comment (dgarcia360) [inline src/service/transaction.service.ts]: Could you try to fix it without using try/catch? Thanks!

Comment (NemTechCN) [reply src/service/transaction.service.ts]: ok,

File: src/service/transaction.service.ts [hunk 2]
```diff
@@ -51,7 +51,12 @@ export class TransactionService {
             } else {
                 transactionFormatted += transaction.recipient.toHex();
             }
-            transactionFormatted += transaction.message.payload.length > 0 ? ' Message:\"' + transaction.message.payload + '\"' : '';
+            transactionFormatted += '';
+            if (transaction.message.payload != undefined){
```
Comment (dgarcia360) [inline src/service/transaction.service.ts]: @rg911 Do you know of a better way to evaluate this in typescript? I don't like much using undefined.

Comment (rg911) [reply src/service/transaction.service.ts]: You shouldn't expecting `undefined here. Instead, if message is empty, SDK returns `EmtpyMessage` object.

**Comment by dgarcia360** [comment]
Closed in favour of #56

**Pull request closed without merging**